src/pages/orders/edit-ticket-page.js

```diff
@@ -198,11 +199,22 @@ class EditTicketPage extends React.Component {
         const { refundAmount, refundNotes} = this.state;
         const { refundTicket, entity} = this.props;
 
-        if(parseFloat(refundAmount) > 0 &&  parseFloat(refundAmount) <= parseFloat(entity.final_amount)) {
+        const maxAllowedAmount2Refund = entity.net_selling_cost - entity.refunded_amount;
+
+        if (refundAmount > maxAllowedAmount2Refund) {
+            this.setState({...this.state,
+                refundAmountError: true
+            });
+            return
+        }
+        
+        if( parseFloat(refundAmount) > 0 && 
+            parseFloat(refundAmount) <= parseFloat(entity.final_amount)) {
```

Comment:
i dont think this validation is correct
 parseFloat(refundAmount) <= parseFloat(entity.final_amount) should be removed